mindsdb_sql/parser/dialects/mindsdb/parser.py

```diff
@@ -50,7 +50,7 @@ class MindsDBParser(Parser):
        'use',
        'describe',
        'create_predictor',
-       'datasource_engine',
+       'database_engine',
```

Comment:
It think this is not need here. This is output command and we don't need to support just 'DATABASE id'

Reply:
deleted